azure-pipelines.yml

```diff
@@ -3,24 +3,43 @@ jobs:
     displayName: Ubuntu-16.04
     pool:
       vmImage: ubuntu-16.04
```

Comment:
I bumped this to 18.04, so may have caused some merge conflicts for you here.  Sorry!